app/serializers/server_room_serializer.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
 
 class ServerRoomSerializer < RoomSerializer
-  attributes :owner, :status, :participants
+  attributes :owner, :status, :participants, :last_session
 
   def owner
     object.user.name
   end
+
+  def last_session
+    return 'No meeting yet' if object.last_session.nil?
```

Comment:
I think we shouldn't be preparing the representation of our data in the backend.
So, `last_session` should have for example `nil|datetime` or `nil|integer`.
The frontend should be the only entity concerned with mapping these values to its adequate represenatation.
If it's falssy then the client app  will show **No meeting yet** and do any I18n otherwise it will be in charge to display the format of the date and time according to the user browser timezone and language (upcoming).

Reply:
I agree.
We should talk about this next meeting.